package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "repository": {},
+  "license": "MIT",
+  "scripts": {
+    "deploy": "brunch build --production",
+    "watch": "brunch watch --stdin",
+    "postinstall": "cp node_modules/tachyons/css/tachyons.min.css web/static/css/"
+  },
+  "dependencies": {
+    "phoenix": "file:deps/phoenix",
+    "phoenix_html": "file:deps/phoenix_html",
+    "tachyons": "^4.6.2"
+  },
+  "devDependencies": {
+    "babel-brunch": "~6.0.0",
```

Comment:
unless we are _using_ ES6/7 in the project this devDependency is downloading 200mb+ of data. because https://www.npmjs.com/package/babel-preset-latest is _absolutely_ _humungous_!
This means that CI runs slower and anyone on a device that has less space (_e.g. Chromebook_) is discriminated against.